HopsanGUI/Dialogs/OptimizationScriptWizard.cpp

```diff
@@ -0,0 +1,1611 @@
+/*-----------------------------------------------------------------------------
+
+ Copyright 2017 Hopsan Group
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+ The full license is available in the file GPLv3.
+ For details about the 'Hopsan Group' or information about Authors and
+ Contributors see the HOPSANGROUP and AUTHORS files that are located in
+ the Hopsan source code root directory.
+
+-----------------------------------------------------------------------------*/
+
+//!
+//! @file   OptimizationScriptWizard.cpp
+//! @author Robert Braun <robert.braun@liu.se>
+//! @date   2018-05-25
+//!
+//! @brief Contains a class for the optimization script wizard
+//!
+//$Id$
+
+//Qt includes
+#include <QDebug>
+#include <QMessageBox>
+#include <QFileDialog>
+#include <QLineEdit>
+#include <QPushButton>
+#include <QTimer>
+#include <QScrollArea>
+#include <QToolBar>
+
+#include "global.h"
+
+//Hopsan includes
+#include "Configuration.h"
+#include "DesktopHandler.h"
+#include "Dialogs/OptimizationDialog.h"
+#include "global.h"
+#include "GUIObjects/GUISystem.h"
+#include "GUIPort.h"
+#include "HcomHandler.h"
+#include "Utilities/HelpPopUpWidget.h"
+#include "ModelHandler.h"
+#include "OptimizationHandler.h"
+#include "Widgets/HcomWidget.h"
+#include "Utilities/GUIUtilities.h"
+#include "OptimizationScriptWizard.h"
+
+//! @brief Constructor
+OptimizationScriptWizard::OptimizationScriptWizard(SystemContainer* pSystem, QWidget *parent)
+    : QWizard(parent)
+{
+        //Set the name and size of the main window
+    this->resize(800,600);
```

Comment:
Maybe scale to percentage of curent resolution if someone havs super high resolution